core/src/scss/components/alert/_alert.scss

```diff
@@ -172,6 +187,21 @@
 // Optional open footer area.
 // -----------------------------------------------------------------------------
 .su-alert__footer {
+  &.su-link-black-true {
+    a {
+      &:hover {
+        background: $su-color-digital-blue-light;
+      }
+    }
+  }
+  &.su-link-black-false {
+    a {
+      &:hover {
+        background: $su-color-white;
+        color: $su-color-black;
+      }
+    }
+  }
```

Comment:
We don't need to worry about the CTA link in the alert footer so please remove these changes.